package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "start": "node server.js",
-    "test": "mocha -r ts-node/register -r test-setup.js src/ts/**/*.spec.tsx"
+    "test": "npm run lint && mocha -r ts-node/register -r test-setup.js src/ts/**/*.spec.tsx",
```

Comment:
we should most likely split this up into more tasks. The linter should check within the build process, not necessarily the tests, but still this is a step forward.